File: Assignment_PH36760_API/models/billdetails.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const BillDetails = new Schema({
    billId : {type: String},
    productId: {type: String},
    quantity: {type: Number},
},{
    timestamps: true
});

module.exports = mongoose.model('billdetails', BillDetails);

File: Phong_PH36760_Assignment/app/src/main/java/com/example/phong_assignment/Model/BillDetails.java
package com.example.phong_assignment.Model;

public class BillDetails {
    private String billId, productId;
    private int quantity;

    public BillDetails(String billId, String productId, int quantity) {
        this.billId = billId;
        this.productId = productId;
        this.quantity = quantity;
    }

    public BillDetails() {
    }

    public String getBillId() {
        return billId;
    }

    public void setBillId(String billId) {
        this.billId = billId;
    }

    public String getProductId() {
        return productId;
    }

    public void setProductId(String productId) {
        this.productId = productId;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}
